integration/pom.xml

```diff
@@ -57,6 +57,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.mule.sdk</groupId>
+            <artifactId>mule-sdk-api</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
```

Comment:
is the version necessary? don't we have dependency management on this? If DM not available here, then at least use a property

Reply:
not necessary, my bad, I will remove it